### PR TITLE
Bump VSEditor to latest 17.2

### PR DESCRIPTION
**Pull request by BretJohnson**

### CODE REVIEW

**Review by etvorun — APPROVED** [review]

**Review by mgoertz-msft — APPROVED** [review]
Never heard of `MiniEditor` before but change looks fine.

File: NuGet.Config [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+	<packageSources>
+		<!-- Don't use any sources set by the system-->
+		<clear />
+		<add key="vssdk" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vssdk/nuget/v3/index.json" />
```
Comment (sandyarmstrong) [inline NuGet.Config:6]: You'll still need nuget.org as well.

File: Microsoft.VisualStudio.MiniEditor/Microsoft.VisualStudio.MiniEditor.csproj [hunk 2]
```diff
@@ -39,7 +39,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.VisualStudio.Composition" Version="16.9.20" />
-    <PackageReference Include="Microsoft.VisualStudio.Threading" Version="17.0.32-alpha" />
+    <PackageReference Include="Microsoft.VisualStudio.Threading" Version="17.3.1-alpha" />
```
Comment (sandyarmstrong) [inline Microsoft.VisualStudio.MiniEditor/Microsoft.VisualStudio.MiniEditor.csproj:42]: Oh, maybe make this match the version used by the editor?

Comment (sandyarmstrong) [reply Microsoft.VisualStudio.MiniEditor/Microsoft.VisualStudio.MiniEditor.csproj:42]: `17.2.20-alpha` or something like that

Comment (BretJohnson) [reply Microsoft.VisualStudio.MiniEditor/Microsoft.VisualStudio.MiniEditor.csproj:42]: Yeah, that's probably better. Updated here: https://github.com/xamarin/MiniEditor/pull/5

**Pull request merged**